onnxruntime/python/tools/pytorch_export_contrib_ops.py

```diff
@@ -65,8 +70,10 @@ def inverse(g, self):
         return g.op("com.microsoft::Inverse", self).setType(self.type())
     _reg(inverse)
 
-    def gelu(g, self):
-        return g.op("com.microsoft::Gelu", self).setType(self.type())
+    def gelu(g, self: torch._C.Value, approximate: str = "none"):
+        if approximate == "none":
```

Comment:
Please add a comment explaining this.
symbolic_opset9.gelu supports `approximate="none"`, so it's not at all obvious why this is here.

https://github.com/pytorch/pytorch/blob/f4200600e44d8696134b14aac142cfc20b276e36/torch/onnx/symbolic_opset9.py#L3335

Reply:
Added. onnxruntime implements a custom op for gelu so we respect it (https://github.com/microsoft/onnxruntime/blob/4e9e01cb3c008335a2471c27dbdf7dd5d12e4224/onnxruntime/contrib_ops/cpu/activations.h#L56). Do this seem to be the right thing to do to you?

Reply:
I suggest a comment like:
```python
# Use microsoft::Gelu for performance if possible. It only supports approximate = "none"
```
And move the comment above the if.

Reply:
Thanks! Done